### PR TITLE
Akshara

### PR DESCRIPTION
**Pull request by Akshara789**
Registration Frontend. I added Password reset frontend. Also I made changes to files according to the review.

### CODE REVIEW

File: diagramming-tool/package-lock.json
Comment (AnushiyaM11) [inline diagramming-tool/package-lock.json:1]: Please ignore the package-lock.json file while committing changes to the branch

File: diagramming-tool/src/App.js [hunk 1]
```diff
@@ -1,5 +1,15 @@
 import './App.css';
 import Registration from './Component/Register/Registration';
+
+
+function App() {
+  return (
+    <div className="App">
+      <header className="App-header">
+      <Registration/>
```
Comment (AnushiyaM11) [inline diagramming-tool/src/App.js:9]: Registration component should be called in the Routes

File: diagramming-tool/src/Component/Register/.babelrc
Comment (AnushiyaM11) [inline diagramming-tool/src/Component/Register/.babelrc:1]: .babelrc file should be included in the root folder not within the component folder

File: diagramming-tool/src/Component/Register/Registration.jsx [hunk 2]
```diff
@@ -25,6 +27,37 @@ const Registration = () => {
       [name]: value
     });
 
+
+   
+    if (name === 'firstname') {
```
Comment (AnushiyaM11) [inline diagramming-tool/src/Component/Register/Registration.jsx:32]: The if else statements are redundant

File: diagramming-tool/src/Component/Register/Registration.jsx
Comment (AnushiyaM11) [inline diagramming-tool/src/Component/Register/Registration.jsx:1]: Please do check for the changes after merging

File: diagramming-tool/src/Component/Register/jest.config.js
Comment (AnushiyaM11) [inline diagramming-tool/src/Component/Register/jest.config.js:1]: jest.config.js files should be included in the root folder not within the component folder

**Review by AnushiyaM11 — CHANGES_REQUESTED** [review]
Please do the needful changes

File: diagramming-tool/package-lock.json
Comment (Akshara789) [reply diagramming-tool/package-lock.json:1]: Done changes

File: diagramming-tool/src/App.js (hunk 1 above)
Comment (Akshara789) [reply diagramming-tool/src/App.js]: Done changes

File: diagramming-tool/src/Component/Register/.babelrc
Comment (Akshara789) [reply diagramming-tool/src/Component/Register/.babelrc:1]: Done changes

File: diagramming-tool/src/Component/Register/Registration.jsx (hunk 2 above)
Comment (Akshara789) [reply diagramming-tool/src/Component/Register/Registration.jsx]: Done changes

File: diagramming-tool/src/Component/Register/jest.config.js
Comment (Akshara789) [reply diagramming-tool/src/Component/Register/jest.config.js:1]: Done changes

**Pull request closed without merging**